crates/forge/src/scarb_test_collector.rs

```diff
@@ -0,0 +1,16 @@
+use crate::collecting::CompiledTestCrateRaw;
```

Comment:
Either move `build_test_artifacts_with_scarb` here or move this to the `scarb` module